src/imagej/stack.py

```diff
@@ -29,6 +29,7 @@ def rai_slice(rai, imin: Tuple, imax: Tuple, istep: Tuple):
     #   >>> help(JLong)
     #
     # So while the import here is unfortunate, it avoids the issue.
+    # TODO: Change to scyjava.new_jarray once we have that function.
```

Comment:
Shall we make an issue to track this?

Reply:
🤷 Even after the `scyjava.new_jarray` function is merged, it's not urgent to update the downstream code. I see it more as a "when I stumble over this again" type change. So an issue is fine, but just adds an issue to the pile we have to sort through whenever we are reviewing issues.

Reply:
Alright, well, if you don't care, then I don't feel strongly enough about needing a new issue.